hubtools/src/lib.rs

```diff
@@ -618,6 +675,7 @@ impl RawHubrisArchive {
     pub fn write_version_to_caboose(
         &mut self,
         version: &str,
+        epoch: Option<&String>,
```

Comment:
Why not use an `Option<u32>` here?

Reply:
Also, `epoch` does not appear to be used in the caboose here; it's only written to the image header?

Reply:
changed to u32 and it gets written to the caboose.